README.md

```diff
@@ -1,6 +1,11 @@
 # registers-cdap-service
 
-Currently contains one service for retrieving SIC descriptions for codes
+Currently contains Two services:
+ - Sic07Service retrieving SIC descriptions for codes
+ - CHService
```

Comment:
Is it worth renaming 'CH' everywhere to CompanyHouse? So for example, `CompanyHouseService.java` , `CompanyHouseServiceTest.java`.  A bit more verbose, but more readable immediately without having to delve through the README.

Reply:
@matthewandrewpalmer , I realise that this is quite a disruptive change. Feel free to disregard if @willis7 is happy with the current names.

Reply:
There is a lot to change but It does make it a lot more readable, so I've made the change now.